ci/docker_sources/hil_container/hil.cfg

```diff
@@ -0,0 +1,36 @@
+[general]
+log_level = debug
+
+[auth]
+require_authentication = False
+
+[headnode]
+trunk_nic = eth0
+base_imgs = img1, img2, img3, img4
+libvirt_endpoint = qemu:///system
+
+[client]
+endpoint = http://127.0.0.1:7000
+
+[database]
+uri = sqlite:////home/hil/hil.db
+#uri = postgresql://<username>:<password>@<address>/<dbname>
+
+[devel]
+dry_run=True
+
+[extensions]
+hil.ext.switches.mock =
+hil.ext.obm.mock =
+
+#hil.ext.network_allocators.null =
+
+# Depending on what you're trying to do, you may want to use the vlan_pool
+# network allocator instead of the null allocator. To do this, comment out the
+# null allocator extension above, and uncomment the following:
+#
+hil.ext.auth.null =
```

Comment:
I think BMI works with database authentication, in fact there are some tests in BMI that check for that.

Reply:
you are right, we would have to bootstrap the database somewhere.